package.json

```diff
@@ -47,5 +47,8 @@
 		"prettier-plugin-organize-imports": "4.1.0",
 		"prettier-plugin-tailwindcss": "0.6.8",
 		"typescript": "^5.6.2"
+	},
+	"dependencies": {
```

Comment:
This should be installed in the `ui/packages.json` file and not in the global one.